workers/unity/Packages/com.improbable.gdk.tools/GdkToolsConfigurationEditorExtensions.cs

```diff
@@ -0,0 +1,154 @@
+﻿using System;
+using System.Collections.Generic;
+using UnityEditor;
+using UnityEngine;
+using Object = UnityEngine.Object;
+
+namespace Improbable.Gdk.Tools
+{
+    /// <summary>
+    ///     Defines a custom inspector window that allows you to configure the GDK Tools.
+    /// </summary>
+    [CustomEditor(typeof(ScriptableGdkToolsConfiguration))]
+    public class GdkToolsConfigurationInspector : Editor
+    {
+        internal const string SchemaStdLibDirLabel = "Schema Standard Library Directory";
```

Comment:
Drop the "Title Case" from these labels, e.g. "Schema standard library directory"